### PR TITLE
Devop-197-hadolint-core-frontend

### PR DESCRIPTION
**Pull request by DaisyDomergue**

### CODE REVIEW

**Comment by DaisyDomergue** [comment]
@kare this is working correctly but the lining is not passing.

**Comment by kare** [comment]
@DaisyDomergue You can fix lint by removing curl installation from `Dockerfile`. It's included in the base image by default.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,6 +23,11 @@ jobs:
         run: |
           npm ci
       # job specific steps
+      - name: Hadolint for linting Dockerfile
+        uses: hadolint/hadolint-action@v1.6.0
+        with:
+          dockerfile: ./app/Dockerfile
+          config: ./.hadolint.yaml
```
Comment (kare) [inline .github/workflows/ci.yml:30]: Move Hadolint after line 34 `npm run styleinit`

**Review by kare — APPROVED** [review]
LGTM

**Pull request merged**